server-spi-private/src/main/java/org/keycloak/models/AccountRoles.java

```diff
@@ -29,6 +29,7 @@
     String VIEW_APPLICATIONS = "view-applications";
     String VIEW_CONSENT = "view-consent";
     String MANAGE_CONSENT = "manage-consent";
+    String DELETE_ACCOUNT = "delete-own-account";
```

Comment:
Should just be "delete-account" to make it consistent with other roles. It is "manage-account", not "manage-own-account"